modules/cluster-logging-collector-log-forward-loki.adoc

```diff
@@ -0,0 +1,83 @@
+
+.Procedure
+
+. Create a `ClusterLogForwarder` CR YAML file similar to the following:
++
+[source,yaml]
+----
+apiVersion: "logging.openshift.io/v1"
+kind: ClusterLogForwarder
+metadata:
+  name: instance <1>
+  namespace: openshift-logging <2>
+spec:
+  outputs:
+   - name: loki-insecure <3>
+     type: "loki" <4>
+     url: http://loki.insecure.com:9200 <5>
+   - name: loki-secure
+     type: "loki"
+     url: https://loki.secure.com:9200 <6>
+     secret:
+        name: loki-secret <7>
+  pipelines:
+   - name: application-logs <8>
+     inputRefs: <9>
+     - application
+     - audit
+     outputRefs:
+     - loki-secure <10>
+     labels:
+       myLabel: "myValue" <11>
+   - name: infrastructure-audit-logs <12>
+     inputRefs:
+     - infrastructure
+     outputRefs:
+     - loki-insecure
+     labels:
+       logs: "audit-infra"
+----
+<1> The name of the `ClusterLogForwarder` CR must be `instance`.
+<2> The namespace for the `ClusterLogForwarder` CR must be `openshift-logging`.
+<3> Specify a name for the output.
+<4> Specify the `loki` type.
+<5> Specify the URL and port of the Loki system as a valid absolute URL. You can use the `http` (insecure) or `https` (secure HTTP) protocol. If the cluster-wide proxy using the CIDR annotation is enabled, the output must be a server name or FQDN, not an IP Address.
+<6> For a secure connection, you can specify an `https` or `http` URL that you authenticate by specifying a `secret`.
+<7> For an `https` prefix, specify the name of the secret required by the endpoint for TLS communication. The secret must exist in the `openshift-logging` project, and must have keys of: *tls.crt*, *tls.key*, and *ca-bundle.crt* that point to the respective certificates that they represent. Otherwise, for `http` and `https` prefixes, you can specify a secret that contains a username and password. For more information, see the following "Example: Setting secret that contains a username and password."
+<8> Optional: Specify a name for the pipeline.
+<9> Specify which log types to forward by using the pipeline: `application,` `infrastructure`, or `audit`.
+<10> Specify the name of the output to use when forwarding logs with this pipeline.
+<11> Optional: String. One or more labels to add to the logs.
+<12> Optional: Configure multiple outputs to forward logs to other external log aggregators of any supported type:
+** A name to describe the pipeline.
+** The `inputRefs` is the log type to forward by using the pipeline: `application,` `infrastructure`, or `audit`.
+** The `outputRefs` is the name of the output to use.
+** Optional: String. One or more labels to add to the logs.
+
+. Create the CR object:
++
+[source,terminal]
+----
+$ oc create -f <file-name>.yaml
+----
+
+The Red Hat OpenShift Logging Operator redeploys the Fluentd pods. If the pods do not redeploy, you can delete the Fluentd pods to force them to redeploy.
```

Comment:
Again this is a generic troubleshooting tip not related to Loki, not sure it belongs here since: 
a) it shouldn't happen, we're suggesting the product is buggy and
b) it applies any time you update the forwarder CR for any reason, it's not specific to using Loki, so should be documented in some more general place.

Reply:
I agree with this proposal and am getting feedback from the team. This removal will affect about eight files.

Reply:
Relocated that statement to a troubleshooting section. Thanks.